lava_test_plans/include/test_target.jinja2

```diff
@@ -1,5 +1,11 @@
+{% set use_docker_image_for_test_target = use_docker_image_for_test_target|default(false) %}
+
 {% if enable_tests is defined and enable_tests %}
 - test:
+{% if use_docker_image_for_test_target == true %}
```

Comment:
If this is used in the variable.(ini|yaml) file it should be with uppercase

Reply:
thanks, will update to use uppercase